File: backjoon/greedy/JS/수들의 합.js
const input = require("fs")
  .readFileSync(process.platform === "linux" ? "./dev/stdin" : "./input.txt")
  .toString()
  .trim()
  .split("\n");

const s = +input.shift();
let sum = 0;
let count = 0;
let number = 1;

while (sum + number <= s) {
  sum += number;
  count++;
  number++;
}

console.log(count);

File: backjoon/greedy/Python/수들의 합.py
def solve(N):
   
    sum = 0
    count = 0
    num = 1

    while(sum + num <= N) :
       sum += num
       count += 1
       num += 1


    print(count)

if __name__ == "__main__":
    
    N = int(input())
    
    solve(N)  # N과 Array를 solve 함수에 전달